flink-tests/src/test/java/org/apache/flink/test/checkpointing/UnalignedCheckpointRescaleWithMixedExchangesITCase.java

```diff
@@ -230,7 +230,7 @@ private static JobClient createMultiInputDAG(StreamExecutionEnvironment env) thr
 
         sourceStream1
                 .rebalance()
-                .connect(forwardedStream)
+                .connect(forwardedStream.rebalance())
```

Comment:
It looks like using FORWARD was the intention behind `forwardedStream`:
```
        // Keep the same parallelism to ensure the ForwardPartitioner will be used.
```

But now we want to avoid FORWARD in any case, right?
Would it make sense to rename the variable, change the comment, and add `rebalance` earlier?